zitadel/terraform/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+jwt_profile_file = "local-token.json"
+redirect_xpanse-ui_uris    = ["http://localhost:3000/authentication/callback"]
```

Comment:
same here also. gui_base_url must be in envrionment vars

Reply:
fixed